rel/overlay/etc/default.ini

```diff
@@ -147,6 +147,28 @@ max_db_number_for_dbs_info_req = 100
 ; prevent non-admins from accessing /_all_dbs
 ; admin_only_all_dbs = true
 
+;;;; Moved from [httpd] ;;;;;
+authentication_handlers = {couch_httpd_auth, cookie_authentication_handler}, {couch_httpd_auth, default_authentication_handler}
+secure_rewrites = true
+allow_jsonp = false
+
+enable_cors = false
+enable_xframe_options = false
```

Comment:
I wonder if we could move these to code default and keep them commented out and it would help out with another pending issue https://github.com/apache/couchdb/issues/3473